oak_functions/loader/src/tests.rs

```diff
@@ -30,9 +29,7 @@ use std::{
 extern crate test;
 use test::Bencher;
 
-const TEST_WASM_MODULE_PATH: &str = "testdata/non-oak-minimal.wasm";
 const MANIFEST_PATH: &str = "examples/hello_world/module/Cargo.toml";
```

Comment:
Is it fine that `oak_functions` crate relies on `examples` crate (which it doesn't include) for testing?

Reply:
I think it's fine, otherwise we need yet another example anyways.